nuxeo-drive-client/nxdrive/utils.py

```diff
@@ -297,17 +297,16 @@ def find_resource_dir(directory, default_path):
     return dir_path
 
 
-def force_decode(string, codecs=['utf-8', 'cp1252']):
+def force_decode(string, codecs=('utf-8', 'cp1252')):
```

Comment:
Why tuple instead of list, aren't we dealing with an homogenous sequence of encodings?